src/Compilers/VisualBasic/Portable/Symbols/MethodSymbol.vb

```diff
@@ -141,6 +141,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
         End Function
 
         ''' <summary>
+        ''' Build and add synthesized return type attributes for this method symbol.
+        ''' </summary>
+        Friend Overridable Sub AddSynthesizedReturnTypeAttributes(ByRef attributes As ArrayBuilder(Of SynthesizedAttributeData))
```

Comment:
It seems to be possible to have regular return type attributes and then synthesized ones. Two sets are typically combined, but not always. Sometimes attributes are obtained for diagnostic purposes or to propagate onto something else, so synthetics are not interesting.
AddSynthesizedReturnTypeAttributes is only needed in emit and therefore this separation (I think)


---
In reply to: [79721290](https://github.com/dotnet/roslyn/pull/13923#discussion_r79721290) [](ancestors = 79721290,79711243)